src/releases/ReleasesClient.ts

```diff
@@ -0,0 +1,423 @@
+  }
+
+  /**
+   * An unarchive action is used to restore an archived release and all documents
+   * with the content they had just prior to archiving.
+   *
+   * @param releaseId - The id of the release to unarchive (with no prefix)
+   */
+  unarchive(
+    {releaseId}: {releaseId: string},
+    options?: BaseActionOptions,
+  ): Observable<SingleActionResult> {
+    const unarchiveAction: UnarchiveReleaseAction = {
+      actionType: 'sanity.action.release.unarchive',
+      releaseId,
+    }
+
+    return _action(this.#client, this.#httpRequest, unarchiveAction, options)
+  }
+
+  /**
+   * A schedule action queues a release for publishing at the given future time.
+   * The release is locked such that no documents in the release can be modified and
+   * no documents that it references can be deleted as this would make the publish fail.
+   * At the given time, the same logic as for the publish action is triggered.
+   *
+   * @param releaseId - The id of the release to schedule (with no prefix)
+   */
+  schedule(
+    {releaseId, publishAt}: {releaseId: string; publishAt: string},
+    options?: BaseActionOptions,
+  ): Observable<SingleActionResult> {
+    const scheduleAction: ScheduleReleaseAction = {
+      actionType: 'sanity.action.release.schedule',
+      releaseId,
+      publishAt,
+    }
+
+    return _action(this.#client, this.#httpRequest, scheduleAction, options)
+  }
+
+  /**
+   * An unschedule action is used to stop a release from being published.
+   * The documents in the release are considered unlocked and can be edited again.
+   * This may fail if another release is scheduled to be published after this one and
+   * has a reference to a document created by this one.
+   *
+   * @param releaseId - The id of the release to unschedule (with no prefix)
+   */
+  unschedule(
+    {releaseId}: {releaseId: string},
+    options?: BaseActionOptions,
+  ): Observable<SingleActionResult> {
+    const unscheduleAction: UnscheduleReleaseAction = {
+      actionType: 'sanity.action.release.unschedule',
+      releaseId,
+    }
+
+    return _action(this.#client, this.#httpRequest, unscheduleAction, options)
+  }
+
+  /**
+   * A delete action is used to delete a published or archived release.
+   * The backing system document will be removed from the dataset.
+   *
+   * @param releaseId - The id of the release to delete (with no prefix)
+   */
+  delete(
+    {releaseId}: {releaseId: string},
+    options?: BaseActionOptions,
+  ): Observable<SingleActionResult> {
+    const deleteAction: DeleteReleaseAction = {
+      actionType: 'sanity.action.release.delete',
+      releaseId,
+    }
+
+    return _action(this.#client, this.#httpRequest, deleteAction, options)
+  }
+
+  getDocuments(
+    {releaseId}: {releaseId: string},
+    options?: BaseMutationOptions,
+  ): Observable<RawQueryResponse<SanityDocument[]>> {
+    return _getReleaseDocuments(this.#client, this.#httpRequest, releaseId, options)
+  }
+}
+
+/** @internal */
```

Comment:
This is marked as internal, but it's exposed on `client.releases`. Is that intentional?